relay-server/src/actors/processor.rs

```diff
@@ -763,7 +799,7 @@ impl EnvelopeProcessorService {
     /// are out of range after clock drift correction.
     fn process_sessions(&self, state: &mut ProcessEnvelopeState) {
         let received = state.envelope_context.received_at();
-        let extracted_metrics = &mut state.extracted_metrics;
+        let extracted_metrics = &mut &mut state.extracted_metrics.project_metrics;
```

Comment:
Doesn't this work just with taking one mutable reference? 

```suggestion
        let extracted_metrics = &mut state.extracted_metrics.project_metrics;
```

Reply:
good catch! yeah it works without second `&mut`